plugins/commands/autocomplete/command/install.rb

```diff
@@ -0,0 +1,49 @@
+require 'optparse'
+
+require 'vagrant/util/install_cli_autocomplete'
+
+module VagrantPlugins
+  module CommandAutocomplete
+    module Command
+      class Install < Vagrant.plugin("2", :command)
+        def execute
+          options = {
+            :shells => []
```

Comment:
```ruby
shells: []
```

Just for consistency as that is style that has been most widely adopted throughout the codebase.